Sources/Sentry/SentryScreenshot.m

```diff
@@ -47,7 +47,9 @@ - (void)saveScreenShots:(NSString *)path
 
         if ([window drawViewHierarchyInRect:window.bounds afterScreenUpdates:false]) {
             UIImage *img = UIGraphicsGetImageFromCurrentImageContext();
-            [result addObject:UIImagePNGRepresentation(img)];
+            if (img.size.width > 0 && img.size.height > 0) {
+                [result addObject:UIImagePNGRepresentation(img)];
+            }
```

Comment:
m: It's more likely to have a problem with the image conversion then a screen with size 0;
```suggestion
            if (img.size.width > 0 || img.size.height > 0) {
                NSData* bytes = UIImagePNGRepresentation(img);
                if (bytes && bytes.length > 0) {
                   [result addObject:bytes];
                }
            }
```